### PR TITLE
Subsystem additions and edits

### PR DESCRIPTION
**Pull request by ramenroboticsprogrammer**
Changes include adding subsystems, adding commands, and dividing the ArmIntakeSystem into two separate subsystems.

### CODE REVIEW

File: src/main/java/frc/robot/commands/ArmDefaultCommand.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants.ArmConstants;
+import frc.robot.Constants.OperatorConstants;
+import frc.robot.subsystems.ArmSystem;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+
+public class ArmDefaultCommand extends CommandBase {
+    // Goal is to get joystick input and turn that into what we want to set the arm to
+    private XboxController m_Controller;
+    private ArmSystem m_arm = new ArmSystem();
```
Comment (gsimonnet) [inline src/main/java/frc/robot/commands/ArmDefaultCommand.java:13]: m_arm gets overwritten in the constructor, the new statement here is spurious

File: src/main/java/frc/robot/commands/ArmDefaultCommand.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    public ArmDefaultCommand(ArmSystem arm) {
+        m_arm = arm;
```
Comment (gsimonnet) [inline src/main/java/frc/robot/commands/ArmDefaultCommand.java:16]: addRequirements(m_arm)

**Review by gsimonnet — APPROVED** [review]
A couple of minor issues to fix in a future change

File: src/main/java/frc/robot/commands/ArmDefaultCommand.java (hunk 2 above)
Comment (RamenRobotics9036) [reply src/main/java/frc/robot/commands/ArmDefaultCommand.java:16]: Good point @gsimonnet!  I spoke with David, and he showed me that the issue is fixed in develop.  Let us know if you're OK resolving this comment.

File: src/main/java/frc/robot/commands/ArmDefaultCommand.java (hunk 1 above)
Comment (RamenRobotics9036) [reply src/main/java/frc/robot/commands/ArmDefaultCommand.java:13]: Good point @gsimonnet! I spoke with David, and he showed me that the issue is fixed in develop. Let us know if you're OK resolving this comment.

**Pull request merged**